src/interfaces/assistants_web/src/utils/citations.ts

```diff
@@ -11,6 +11,26 @@ export const fixMarkdownImagesInText = (text: string) => {
 
 const formatter = new Intl.ListFormat('en', { style: 'long', type: 'conjunction' });
 
+export const fixCitationsLeadingMarkdown = (citations: Citation[], originalText: string) => {
+  const citationsCopy = [...citations];
+  const markdownFixList = ['`', '*', '**'];
```

Comment:
we are not covering links, bullet points, etc. Right?

Reply:
Yes, not yet unless we see them